### PR TITLE
whoami: add Brazilian Portuguese translation

### PR DESCRIPTION
**Pull request by schneiderl**

### CODE REVIEW

File: pages.pt-BR/windows/whoami.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# whoami
+
+> Mostra detalhes sobre o usuário atual.
+> Mais informações: <https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/whoami>.
+
+- Mostra o nome de usuário do usário atual:
```
Comment (andrik) [inline pages.pt-BR/windows/whoami.md]:
```suggestion
Mostra o nome de usuário do usuário atual:
```

Comment (andrik) [reply pages.pt-BR/windows/whoami.md]: @schneiderl O que achas de utilizar **login** ou **username** mesmo ao invés de nome de usuário? Sei que estão em inglês mas são termos bem aceitos na comunidade brasileira. Além disso ficaria consistente com o id de logon do último exemplo.

**Comment by andrik** [comment]
@schneiderl  I think the PR title could be changed to follow the [template](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message)

File: pages.pt-BR/windows/whoami.md (hunk 1 above)
Comment (schneiderl) [reply pages.pt-BR/windows/whoami.md]: Faz sentido. Vou fazer a alteração.

Comment (waldyrious) [reply pages.pt-BR/windows/whoami.md]: Careful guys, this change removed the `-` at the start of the line. (How did Travis not complain about this?...)

Comment (schneiderl) [reply pages.pt-BR/windows/whoami.md]: @waldyrious Fixed

**Comment by waldyrious** [comment]
> should we take a look at our bot too?

Yeah, not sure what happened. @sbrl, any thoughts?

File: pages.pt-BR/windows/whoami.md (hunk 1 above)
Comment (mebeim) [reply pages.pt-BR/windows/whoami.md]: @waldyrious tldr-lint does not check pages of all languages:

https://github.com/tldr-pages/tldr/blob/77c60fc861c038d12082367c27851391a23e4e8b/package.json#L14

**Comment by mebeim** [comment]
@waldyrious `tldr-lint` does not check pages of all languages (because of Unicode issues):

https://github.com/tldr-pages/tldr/blob/77c60fc861c038d12082367c27851391a23e4e8b/package.json#L14

**Comment by waldyrious** [comment]
Ah, that explains it. Thanks.

**Comment by sbrl** [comment]
Hrm, tldr-bot is running:

```
 tldr-bot.service - Job that runs the tldr-bot
   Loaded: loaded (/srv/tldr-bot/tldr-bot.service; enabled; vendor preset: enabl
   Active: active (running) since Fri 2019-08-23 00:00:04 BST; 17h ago
 Main PID: 4791 (uwsgi)
   Status: "uWSGI is ready"
    Tasks: 1 (limit: 4657)
   CGroup: /system.slice/tldr-bot.service
           └─4791 /usr/local/bin/uwsgi --socket localhost:6129 --wsgi-file /srv/
l
```

I think we really do need to get around to rebuilding the linter at some point :P

**Comment by mebeim** [comment]
Yep... that's not going to be a trivial work though.

**Comment by sbrl** [comment]
Indeed it isn't :-/
